src/vm/util.cpp

```diff
@@ -2091,8 +2091,8 @@ size_t GetIntelDeterministicCacheEnum()
     unsigned char buffer[16];
 
     DWORD maxCpuid = getextcpuid(0,0,buffer);
-
-    DWORD* dwBuffer = (DWORD*)buffer;
+    DWORD dwBuffer[4];
+    memcpy(dwBuffer, buffer, 16);
```

Comment:
A nit - could you please use ARRAYSIZE(buffer) instead of the hardcoded constant here and at the other place below?